prac_06/guitars.py

```diff
@@ -0,0 +1,35 @@
+"""
+Estimated time to complete  : 45  minutes
+Actual time to complete     : 60  minutes
+"""
+
+from prac_06.guitar import Guitar
+
+
+def main():
+
+    guitars = []
+    count = 0
+
+    print("My guitars!")
+    name = input("Name: ")
+
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+
+        guitars.append(Guitar(name, year, cost))
+
+        print(f"{guitars[count]} added.\n")
```

Comment:
Not sure you really need this print statement or the count variable as it not part of the question. But points to you for practicing using counting.

Reply:
Yeah completely unnecessary but feels satisfying when testing the program to get this feedback haha.